### PR TITLE
[ISSUE#8142] Show time of create topic and subScriptionGroup

### LINKED ISSUE #8142: execution time of create topic and subscription
### Before Creating the Enhancement Request

- [X] I have confirmed that this should be classified as an enhancement rather than a bug/feature.

### Summary

add logs of execution time of create topic and subscription

### Motivation

add logs of execution time of create topic and subscription

### Describe the Solution You'd Like

add logs of execution time of create topic and subscription

### Describe Alternatives You've Considered

/

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by superdev42**
### execution time of create topic and subscription

Fixes #8142

### CODE REVIEW

File: broker/src/main/java/org/apache/rocketmq/broker/processor/AdminBrokerProcessor.java [hunk 1]
```diff
@@ -1462,6 +1468,10 @@ private RemotingCommand updateAndCreateSubscriptionGroup(ChannelHandlerContext c
 
         response.setCode(ResponseCode.SUCCESS);
         response.setRemark(null);
+        long endTime = System.currentTimeMillis();
+        long executionTime = endTime - startTime;
+        LOGGER.info("" + config.getGroupName() + " is " + executionTime + "ms");
```
Comment (guyinyou) [inline broker/src/main/java/org/apache/rocketmq/broker/processor/AdminBrokerProcessor.java:1473]: 这里为什么开头要加一个空字符串？

File: broker/src/main/java/org/apache/rocketmq/broker/filter/CommitLogDispatcherCalcBitMap.java [hunk 2]
```diff
@@ -67,6 +67,7 @@ public void dispatch(DispatchRequest request) {
             while (iterator.hasNext()) {
                 ConsumerFilterData filterData = iterator.next();
 
+
```
Comment (humkum) [inline broker/src/main/java/org/apache/rocketmq/broker/filter/CommitLogDispatcherCalcBitMap.java:70]: No need blank line

File: broker/src/main/java/org/apache/rocketmq/broker/processor/AdminBrokerProcessor.java [hunk 3]
```diff
@@ -1462,6 +1468,10 @@ private RemotingCommand updateAndCreateSubscriptionGroup(ChannelHandlerContext c
...
+        LOGGER.info("" + config.getGroupName() + " is " + executionTime + "ms");
```
Comment (humkum) [inline broker/src/main/java/org/apache/rocketmq/broker/processor/AdminBrokerProcessor.java:1474]: Repeated line

**Review by humkum — CHANGES_REQUESTED** [review]

File: broker/src/main/java/org/apache/rocketmq/broker/processor/AdminBrokerProcessor.java [hunk 4]
```diff
@@ -515,12 +516,15 @@ private synchronized RemotingCommand updateAndCreateTopic(ChannelHandlerContext
             response.setCode(ResponseCode.SYSTEM_ERROR);
             response.setRemark(e.getMessage());
         }
-
+        long endTime = System.currentTimeMillis();
+        long executionTime = endTime - startTime;
+        LOGGER.info("executionTime of create topic:" + topic + " is " + executionTime + "ms");
```
Comment (RongtongJin) [inline broker/src/main/java/org/apache/rocketmq/broker/processor/AdminBrokerProcessor.java:521]: Could you simplify the code？

**Review by guyinyou — APPROVED** [review]
lgtm

**Review by RongtongJin — APPROVED** [review]

**Comment by RongtongJin** [comment]
ping @humkum Do you have any more suggestions？

**Review by Willhow-Gao — APPROVED** [review]

**Review by cserwen — APPROVED** [review]

**Pull request merged**